Modules/ITS/src/ITSFeeTask.cxx

```diff
@@ -149,22 +218,14 @@ void ITSFeeTask::monitorData(o2::framework::ProcessingContext& ctx)
   end = std::chrono::high_resolution_clock::now();
   difference = std::chrono::duration_cast<std::chrono::microseconds>(end - start).count();
   ILOG(Info) << "Processing time: " << difference << ", and TF ID == " << mTimeFrameId << ENDM;
+  mProcessingTime->SetBinContent(mTimeFrameId, difference);
 }
 
-void ITSFeeTask::getEnableLayers()
+void ITSFeeTask::getRunNumber()
 {
-  std::ifstream configFile("Config/ConfigFakeRateFee.dat");
-  for (int ilayer = 0; ilayer < NLayer; ilayer++) {
-    configFile >> mEnableLayers[ilayer];
-    if (mEnableLayers[ilayer]) {
-      ILOG(Info) << "Enable layer : " << ilayer << ENDM;
-    }
-  }
+  std::ifstream configFile("Config/ConfigFee.dat");
```

Comment:
Could you make it slightly more resistant to the very probable case when the file is not there ?

Reply:
Added the checking for the config file and run number file during the commissioning.